package.json

```diff
@@ -5,16 +5,19 @@
   "main": "index.js",
   "scripts": {
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "test": "set NODE_ENV=test&& nyc mocha --require @babel/register 'src/**/**.test.js' --no-timeout ",
+    "test": "set NODE_ENV=test&& npx sequelize db:migrate:undo:all && npm run migrate && npm run seed-db && nyc mocha --require @babel/register 'src/**/**.test.js' --no-timeout --exit ",
     "dev-start": "nodemon --exec babel-node src/index.js",
     "build": "babel src --out-dir build",
     "start": "node build/index.js",
     "drop": "sequelize-cli db:migrate:undo:all",
-    "migrate": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize db:migrate"
+    "seed-db": "npx sequelize db:seed:all",
+    "migrate": "npx sequelize db:migrate"
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
+    "@babel/polyfill": "^7.4.4",
```

Comment:
Please add line 19 `"@babel/polyfill": "^7.4.4",` to the devDependencies.

Reply:
Babel polyfill is a dependency because of the runtime generator we are using which is 'async' and 'await'